stablehlo/reference/ProcessGrid.cpp

```diff
@@ -223,16 +225,19 @@ SmallVector<Tensor> ProcessGrid::recv(ChannelId channelId,
 RendezvousResult ProcessGrid::rendezvous(ProcessGroup processGroup,
                                          ChannelId channelId,
                                          ProcessId processId,
-                                         const Tensor &operand) {
+                                         const ArrayRef<Tensor> operands) {
```

Comment:
avoid `const` ? 
`ArrayRef<T>` is non-owning and immutable: It doesn't manage the lifetime of the underlying data + we cannot modify the data is encapsulates.